src/Admin/EventGroupCategoryAdmin.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use AppBundle\Entity\BaseEventCategory;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class EventGroupCategoryAdmin extends EventCategoryAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->add('name', TextType::class, [
+                'label' => 'Nom',
+            ])
+            ->add('slug', null, [
```

Comment:
why do we need to have slug & status fields for this Entity? maybe we can keep it simpler with only name property?